rio_tiler/io/stac.py

```diff
@@ -47,10 +49,17 @@ def fetch(filepath: str) -> Dict:
     if parsed.scheme == "s3":
         bucket = parsed.netloc
         key = parsed.path.strip("/")
-        return json.loads(aws_get_object(bucket, key))
+
+        client = kwargs.get("client")
+        request_pays = kwargs.get(
+            "request_pays", os.environ.get("AWS_REQUEST_PAYER", False),
+        )
```

Comment:
Why should the kwarg be `request_pays` instead of `request_payer`? Seems better to match the AWS key?

Reply:
Or I guess AWS is inconsistent between their shell env and boto3 API's?

Reply:
Wait, `request_pays` is just your wording in https://github.com/cogeotiff/rio-tiler/blob/3db728210994c5ddc5cd22d117626e9ef2e9ba4d/rio_tiler/utils.py#L34? If so it seems better for the public API to be consistent with AWS APIs, in which case we should use `request_payer`?

Reply:
I don't think it's good to change this now!